DYDX-Bots/Slashed_dYdX/src/findings.ts

```diff
@@ -0,0 +1,18 @@
+import { LogDescription, Finding, FindingSeverity, FindingType } from "forta-agent";
+
+export const createFinding = (log: LogDescription) => {
+  return Finding.fromObject({
+    name: "Slash event has occured on dYdX Security Module.",
+    description: "Slashed event was emitted",
```

Comment:
```suggestion
  return Finding.fromObject({
    name: "Slash event has occured on dYdX Safety Module.",
    description: "Slashed event was emitted",
```
Same change is required in the tests file.